openmdao/core/group.py

```diff
@@ -1420,6 +1420,15 @@ def _setup_connections(self):
                                         else:
                                             simple_warning(msg)
 
+                            for i in src_indices:
+                                if len(i.shape) != len(in_shape) and src_indices.shape != in_shape:
+                                    msg = f"{self.msginfo}: src_indices shape {src_indices.shape} does not match {abs_in} shape " + \
```

Comment:
I don't understand the loop over src_indices.  Can't you just check if src_indices.shape != in_shape?

Reply:
Removed